balloc/balloc.c

```diff
@@ -156,6 +156,11 @@ static bool is_spare(uint64_t alloc_flags);
 static bool is_normal(uint64_t alloc_flags);
 static bool is_any(uint64_t alloc_flag);
 
+static int index_hash(struct m0_balloc_super_block *sb,
+		      m0_bcount_t                   ext_end)
```

Comment:
`m0_bcount_t `and `m0_bindex_t `both are `uint64_t`.
But still use of `m0_bindex_t  `data type would be more appropriate as `ext_end `refers block index, not block count.

Reply:
Done.